### ISSUE TITLE
Large requests broken in 0.2+

### ISSUE THREAD

**Issue by x052bb** [ClickHouse/clickhouse-jdbc#412]
After introducing "write" method in version 0.2, the following issue is relevant again: https://github.com/ClickHouse/clickhouse-jdbc/issues/222

The reason is that all the query text is sent within URL and if query size is large enough you get "413 Request Entity Too Large"

**Comment by miha-g** [comment]
I ran into this issue while trying to insert into a table with large number of columns.
I'm basically running "textbook" example from README:
```
ClickHouseStatement sth = connection.createStatement();
sth
    .write()
    .sql("INSERT INTO my_db.lots_of_columns (>> 1000 columns here <<)")
    .data(new MyCustomInputStream(), ClickHouseFormat.CSV)
    .send();
```
Is there a solution or workaround for this limitation/issue at the moment?

**Comment by mcgrawia** [comment]
We are also running into this issue again as well. Does anyone what would be required for a fix?

**Comment by Qsimple** [comment]
+1

**Comment by zhicwu** [comment]
Since version `0.3.1` both query and external data will be put into request body. Have you tried latest driver? And what's the version of ClickHouse you're using?

Would be great if you can share reproduce steps so that I can fix.

**Comment by x052bb** [comment]
This issues looks fixed in `0.3.1`

**Comment by zhicwu** [comment]
Feel free to re-open if you think the issue still exists in new release.
